docs/environment.yaml

```
@@ -0,0 +1,15 @@
+name: mila-docs
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - sphinx>=4.0.2
+  - recommonmark>=0.7.1
+  - myst-parser>=0.14.0
+  - sphinx-copybutton>=0.3.1
+  - sphinx-prompt>=1.4.0
+  - sphinx_rtd_theme>=0.5.2
+  - pip=22.0
```

should this be
```suggestion
  - pip>=22.0
```

to allow installation of the most up to date version of pip?